src/Native/Unix/System.Security.Cryptography.Native/openssl.c

```diff
@@ -1082,6 +1082,38 @@ int32_t CryptoNative_SetX509ChainVerifyTime(X509_STORE_CTX* ctx,
     return 1;
 }
 
+int32_t CryptoNative_X509StoreSetVerifyTime(X509_STORE* ctx,
```

Comment:
I need to do a pass to remove all no longer called functions from both the Interop.cs files and the native side; but I wanted to defer that until after getting some initial feedback (in case it "un-unuses" some of them)